Dockerfile

```diff
@@ -1,12 +1,12 @@
 ############# builder
-FROM golang:1.15.5 AS builder
+FROM eu.gcr.io/gardener-project/3rd/golang:1.15.5 AS builder
```

Comment:
You will not be able to release the extension with this image because the release job will fail here https://github.com/gardener/gardener-extension-shoot-cert-service/blob/5c60e8535cb3d6eb9557496aace883129f59292c/vendor/github.com/gardener/gardener/hack/.ci/prepare_release#L38

You have to vendor https://github.com/gardener/gardener/pull/3189

Reply:
Thanks, very observant!